components/Charts/TransactionVolume.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from "react";
+import mean from "ramda/src/mean";
+
+import GeneralChart, { GeneralChartProps }  from "./GeneralChart";
+import ChartBox from "./ChartBox";
+import Metric from "types/domain/Metric";
+
+const TransactionVolume: FC<Pick<GeneralChartProps, "data">> = ({ data }) => {
+  const valueAccessor = (d: Metric) => d.transactions_count;
+
+  return (
+    <ChartBox
+      header="Daily Transaction Volume"
+      average={Math.ceil(mean(data.map(valueAccessor)))}
```

Comment:
If this is too much, I'm fine with not doing it, but since I see the same pattern being used in the next component, you can define this as a pointfree function like (all functions come from `ramda`):
```ts
const getAverageWithAccessor = curry((accessor, x) => pipe(
  map(accessor),
  mean,
  Math.ceil
)(x)
```
and then you can replace this line with `getAverageWithAccessor(valueAccessor, data)`